tools/docker-compose/supervisor.conf

```diff
@@ -5,79 +5,75 @@ nodaemon=true
 
 [program:awx-dispatcher]
 command = make dispatcher
-autostart = true
 autorestart = true
-stopwaitsecs = 1
-stopsignal=KILL
+startsecs = 30
```

Comment:
Looks like the default for `stopwaitsecs` is 10. This would be my preference.

The dispatcher with running jobs may not respond well to SIGTERM right now. This is something I packed into https://github.com/ansible/awx/pull/11745, along with a supervisor config change which is almost identical to the change you make here.

But that's getting kicked out until next release. If you want, I could look into a targeted change to _just_ process SIGTERM signals to cancel a job so that it's not left running orphaned on the execution node.